test/AWS.Deploy.CLI.IntegrationTests/SaveCdkDeploymentProject/CustomRecipeLocatorTests.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.IO;
+using AWS.Deploy.CLI.Utilities;
+using AWS.Deploy.Common.DeploymentManifest;
+using AWS.Deploy.Common.IO;
+using AWS.Deploy.Orchestration;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+using Should;
+using System;
+using AWS.Deploy.Common;
+
+namespace AWS.Deploy.CLI.IntegrationTests.SaveCdkDeploymentProject
+{
+    [Collection("SaveCdkDeploymentProjectTests")]
```

Comment:
What is the reason behind using the same collection for `SaveCdkDeploymentProjectTests`, `CustomRecipeLocatorTests` and `RecommendationTests`.

We must be able to run all these test classes in parallel without any dependencies.

Reply:
Discussed offline, it will be addressed in follow-up.

Not ship blocking.